feg/gateway/tools/s8_cli/main_test.go

```diff
@@ -13,5 +13,8 @@ func TestS8Cli(t *testing.T) {
 	// replace the arguments
 	os.Args = []string{"s8_cli", "cs", "-test", "-delete", "0"}
 	fmt.Printf("\ncommand to run: %+v\n", os.Args)
-	main()
+	exitCode := run()
+	if exitCode != 0 {
+		t.Fatalf("expected exit code to be 0, got: %d", exitCode)
+	}
```

Comment:
I don't remember now but I think there was a reason why we were just exiting. Maybe there are some notes on documentation

In any case, this is out of scope of this PR. If this needs to be changed lets create a different PR

Reply:
Ok, sure, I agree that if it was done so intentionally, than this fix should be done in different PR. `make test` just fails when I'm running it in `feg/gateway`, so I thought that should help